host.go

```diff
@@ -360,10 +361,17 @@ func (h *Host) Create(name string) error {
 		return err
 	}
 
+	// create the instance
 	if err := h.Driver.Create(); err != nil {
 		return err
 	}
 
+	// install docker
+	if err := h.Provision(); err != nil {
```

Comment:
The provision step has this somewhat (it checks for `/usr/bin/docker`).  The reason these things were removed (there was nothing in ec2 fwiw) was to try and "standardize" on the ux across drivers.  Once we get things refactored (store, host, etc) this will be a lot easier to implement.